packages/terra-application-menu-layout/src/ApplicationMenuLayout.scss

```diff
@@ -0,0 +1,48 @@
+/* stylelint-disable selector-max-compound-selectors */
+/* stylelint-disable max-nesting-depth */
```

Comment:
I'm seeing this max-nesting-depth come up a lot in our reviews. Perhaps it's worth it to ignore it in our stylelint config, or increase the depth potentially?

Reply:
+1 to this